jenkins/test_orchestrator/JenkinsFile

```diff
@@ -0,0 +1,62 @@
+pipeline {
+    agent any
+
+    parameters {
+        string(name: 's3_bucket'),
+        string(name: 'opensearch_version'),
+        string(name: 'build_id'),
+        string(name: 'architecture')
+    }
+
+    stages {
+        stage('Execute Tests') {
+            steps {
+                // To run all three test jobs in parallel, we use "parallel" step
+                // https://jenkins.io/doc/pipeline/examples/#jobs-in-parallel
+                parallel (
+                    IntegTests: {
+                        build job: 'integ-tests'
+                        parameters: [
+                            string(name: 's3_bucket', value: ${params.s3_bucket}),
+                            string(name: 'opensearch_version', value: ${params.opensearch_version}),
+                            string(name: 'build_id', value: ${params.build_id}),
+                            string(name: 'architecture', value: ${params.architecture})
+                        ]
+                    },
+                    PerfTests: {
+                        build job: 'perf-tests',
+                        parameters: [
+                            string(name: 's3_bucket', value: ${params.s3_bucket}),
+                            string(name: 'opensearch_version', value: ${params.opensearch_version}),
+                            string(name: 'build_id', value: ${params.build_id}),
+                            string(name: 'architecture', value: ${params.architecture})
+                        ]
+                    },
+                    BwcTests: {
+                        build job: 'bwc-tests',
+                        parameters: [
+                            string(name: 's3_bucket', value: ${params.s3_bucket}),
+                            string(name: 'opensearch_version', value: ${params.opensearch_version}),
+                            string(name: 'build_id', value: ${params.build_id}),
+                            string(name: 'architecture', value: ${params.architecture})
+                        ]
+                    },
+                    failFast: false)
+            }
+        }
+
+        stage('ResultPublisher') {
```

Comment:
will remove this stage as it is already handled inside the python tool at per test case level

Reply:
We might want to eventually consolidate all the results across test suites so that its readable for a single run. But again that P1/P2.